src/index.ts

```diff
@@ -0,0 +1,566 @@
+    asm: string
+    hex: string
+  }
+  txinwitness?: string[]
+  sequence: number
+}
+
+export type TxInForCreateRaw = {
+  txid: string
+  vout: number
+  sequence?: number
+}
+
+export type TxOut = {
+  value: number
+  n: number
+  scriptPubKey: {
+    asm: string
+    hex: string
+    reqSigs: number
+    type: scriptPubkeyType
+    addresses: string[]
+  }
+}
+
+export type TxOutForCreateRaw = {
+  address: string
+  data: string
+}
+
+export type TxOutInBlock = {
+  bestblock: string
+  confirmations: number
+  value: number
+  scriptPubKey: {
+    asm: string
+    hex: string
+    reqSigs: number
+    type: scriptPubkeyType
+    addresses: string[]
+  }
+  coinbase: boolean
+}
+
+export type WalletTransaction = {
+  amount: number
+  fee: number
+  confirmations: number
+  blockhash: string
+  blockindex: number
+  blocktime: number
+  txid: string
+  time: number
+  timereceived: number
+  'bip125-replaceable': 'yes' | 'no' | 'unknown'
+  details: {
+    account: string
+    address: string
+    category: 'send' | 'receive'
+    amount: number
+    label?: string
+    vout: number
+    fee: number
+    abandoned: number
+  }[]
+  hex: string
+}
+
+export type WalletInfo = {
+  walletname: string
+  walletversion: number
+  balance: number
+  unconfirmed_balance: number
+  immature_balance: number
+  txcount: number
+  keypoololdest: number
+  keypoolsize: number
+  keypoolsize_hd_internal: number
+  unlocked_until: number
+  paytxfee: number
+  hdseedid: string
+  private_keys_enabled?: boolean
+  avoid_reuse?: boolean
+  scanning: { duration: number; progress: number } | false
+}
+
+export type scriptPubkeyType = string
+
+export type SigHashType =
+  | 'ALL'
+  | 'NONE'
+  | 'SINGLE'
+  | 'ALL|ANYONECANPAY'
+  | 'NONE|ANYONECANPAY'
+  | 'SINGLE|ANYONECANPAY'
+
+export type SignRawTxResult = {
+  hex: string
+  complete: boolean
+  errors?: {
+    txid: string
+    vout: number
+    scriptSig: string
+    sequence: number
+    error: string
+  }[]
+}
+
+export type ValidateAddressResult = {
+  isvalid: boolean
+  address?: string
+  scriptPubKey?: string
+  ismine?: boolean
+  iswatchonly?: boolean
+  isscript?: boolean
+  script?: string
+  hex?: string
+  addresses?: string[]
+  sigsrequired?: number
+  pubkey?: string
+  iscompressed?: boolean
+  account?: string
+  timestamp?: number
+  hdkeypath?: string
+  hdmasterkeyid?: string
+}
+
+export type ImportMultiRequest = {
+  scriptPubKey: string | { address: string }
+  timestamp: number | 'now'
+  redeemScript?: string
+  pubkeys?: string[]
+  keys?: string[]
+  internal?: boolean
+  watchonly?: boolean
+  label?: string
+}
+
+export type Received = {
+  involvesWatchonly?: boolean
+  account: string
+  amount: number
+  confirmations: number
+  label: string
+}
+
+export type ListUnspentOptions = {
+  minimumAmount: number | string
+  maximumAmount: number | string
+  maximumCount: number | string
+  minimumSumAmount: number | string
+}
+
+export type ReceivedByAccount = Received
```

Comment:
what is the difference with received ? maybe one `export` is enough

Reply:
Yes removed